src/types.ts

```diff
@@ -35,17 +44,55 @@ export type AdvancedParameters = AdvancedParametersBase &
   Omit<IAutocompleteParameters, 'resultsPerSection'>;
 
 export type CioAutocompleteProps = CioClientConfig & {
+  /**
+   * Set to `false` to show suggestions only after a user clears their query,
+   * but not when they initially select the input
+   */
   openOnFocus?: boolean;
+  /**
+   * Function that takes in a Search Suggestion, and returns a url to redirect to
+   */
```

Comment:
> If passed, the "Search Suggestions" rendered as HTML items will be wrapped by an HTML link tag with `href` attribute that redirects to the selected query search results page. (e.g., `<a href='www.mywebsite.com/search?query=[selected_search_suggestion]'>[Search Suggestion]</a>`

I think we can make this a bit clearer. I'm sure you can phrase this better than me but I was thinking something like the above

Reply:
How does this sound? 
```
Transforms a `SearchSuggestion` into the desired URL string to be used when rendering anchor tags
```

Reply:
Looks good. Do you think we can add an example like this?

> <a href='www.mywebsite.com/search?query=[selected_search_suggestion]'>[Search Suggestion]</a>

Reply:
How about something like this?

```
<a href=getSearchResultsUrl([selected_search_suggestion])>[Search Suggestion]</a>
```

Afraid they might get confused at what `getSearchResultsUrl` is supposed to return instead

Reply:
Yes this is better